packages/pn-personagiuridica-webapp/src/component/Deleghe/DelegationsElements.tsx

```diff
@@ -222,7 +222,7 @@ export const Menu: React.FC<Props> = ({ menuType, id, userLogged, row, onAction
         onConfirm={handleConfirmClick}
         onConfirmLabel={confirmLabel}
         onClose={onCloseModal}
-        onCloseLabel={t('button.annulla', { ns: 'common' })}
+        onCloseLabel={t('button.indietro', { ns: 'common' })}
```

Comment:
Unfortunately, you changed the wrong label. The wrong behavior indicated in the issue is still apparent in PG
![image](https://github.com/pagopa/pn-frontend/assets/5519535/8a56f3f2-b088-4014-a3a9-72af72ac6ba0)

On the other hand, there are other modals in which the "no confirmation" button reads "Indietro" while it should read "Annulla" as before
![image](https://github.com/pagopa/pn-frontend/assets/5519535/eb280504-c3ea-4cef-ad94-0d6f67fa402e)

Please rework what's needed so that
- the modal that **asks for a code** reads "Indietro"
- the modals that ask for confirmation (rifiuta / revoca) read "Annulla".

Thanks!

Reply:
fix it

Reply:
done. 

![image](https://github.com/pagopa/pn-frontend/assets/119851973/35e49bdb-f3e9-4b0c-9c9f-546964c3e759)
accept delegation in pg

![image](https://github.com/pagopa/pn-frontend/assets/119851973/afd38903-efc5-4e92-9890-995c795798f6)
reject delegation in pg

![image](https://github.com/pagopa/pn-frontend/assets/119851973/f020240c-c3e4-47cf-a927-f7210c2ff787)
revoke delegation in pg

![image](https://github.com/pagopa/pn-frontend/assets/119851973/95e94b02-64d1-4543-a1f5-0e3b4cbcce12)
accept delegation in pf

![image](https://github.com/pagopa/pn-frontend/assets/119851973/d0d6fa27-ceb3-4acd-b012-b5b842424552)
reject delegation in pf

![image](https://github.com/pagopa/pn-frontend/assets/119851973/ed44cc9a-5c33-43a5-9790-9f4411554401)
revoke delefation in pf

Reply:
wunderbar!